packages/api/src/routers/api/dashboards.ts

```diff
@@ -106,15 +107,15 @@ router.post(
         return res.sendStatus(403);
       }
 
-      const { name, charts, query } = req.body ?? {};
+      const { name, charts, query, tags } = req.body ?? {};
```

Comment:
we might want to assert the uniqueness of tagging values and also enforce the size limit

Reply:
good point, added `uniq()`. by size limit do you mean array length or individual tag lengths?

Reply:
yeah I think we should enforce the limit on both lengths tho (zod validator). like individual tag length 32 and array length 50

Reply:
added it for dashboards endpoint. since logViews doesn't have validators, think it'll be better to address it in a separate PR. will add a task